libs/server/Resp/Objects/SetCommands.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Text;
+using System.Xml.Linq;
```

Comment:
where is this used? we try to avoid Linq if possible.

Reply:
It isn't being used anywhere, it was added when I was messing with the set object (Which isn't being used anywhere either).